examples/QuickStart/train.py

```diff
@@ -64,7 +66,7 @@ def main():
 
         batch_obs = np.array(obs_list)
         batch_action = np.array(action_list)
-        batch_reward = calc_discount_norm_reward(reward_list, GAMMA)
```

Comment:
Please remove the file `utils.py` as we are no longer using the function in this file.